taca_ngi_pipeline/deliver/deliver.py

```diff
@@ -457,6 +549,64 @@ def update_delivery_status(self, status="DELIVERED"):
                 if an error occurred when communicating with the database
         """
         return db.update_project(db.dbcon(), self.projectid, delivery_status=status)
+    
+    def deliver_misc_data(self):
```

Comment:
Yes, I dint like it either when I wrote it. But I could not find another way. Even if I create a subclass `ProjectMetaDeliverer` it will still be the same code but in different class. It is because how it works now is that even though a `ProjectDeliverer` is called while delivering a whole project, what actually happens is that `SampleDeliverer.deliver_sample()` is called for each sample. And it is hardcoded in the script to use `files_to_deliver` section form config file (which have the pattern corresponding to Sample data)

Since I am little paranoid, I made redundant code and did not change any native code. I could try do this better but I may have to tweak the original code a bit. Would that be ok @b97pla  ?